mmdet/ops/merge_cells.py

```diff
@@ -0,0 +1,125 @@
+from abc import abstractmethod
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+
+class BaseMergeCell(nn.Module):
+
+    def __init__(self,
+                 channels=256,
+                 binary_op_type='sum',
```

Comment:
List the candidate values in the docstring and check it in the code.

Reply:
I found that I use binary_op_type only for determine the input_channels of out_conv, so I removed it and straightly use in_channels instead.

Reply:
If so, the argument name `in_channels` is misleading. It is not the channels of x1 or x2, we may name it `fused_channels`.

```python
class BaseMergeCell:

    def __init__(self, fused_channels, out_channels, ...):
        pass

class SumCell(BaseMergeCell):

    def __init__(self, in_channels, out_channels, ...):
        super().__init__(in_channels, out_channels, ...)

class ConcatCell(BaseMergeCell):

    def __init__(self, in_channels, out_channels, ...):
        super().__init__(in_channels * 2, out_channels, ...)
```